project-koku/koku
# Remove Source UID OCP cluster-id fallback

Now that the Sources UI can accept a user inputed cluster-id, the source uid fallback can be removed.  If a cluster-id is not found in the `source_ref` field of the `sources` object an error message will be pushed to the platform.

**Testing**
1. Add OCP source without cluster-id added to source_ref field.  Verify status is pushed to platform.
2.  Add OCP Source with cluster-id added to source_ref field

**Test Results**
[ocp_remove_fallback_ut.txt](https://github.com/project-koku/koku/files/3948252/ocp_remove_fallback_ut.txt)
